### PR TITLE
My favorite palindrome

### PR DESCRIPTION
**Pull request by cclauss**
### Describe your change:

* [ ] Add an algorithm?
* [x] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### Checklist:
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [ ] All new Python files are placed inside an existing directory.
* [ ] All filenames are in all lowercase characters with no spaces or dashes.
* [ ] All functions and variable names follow Python naming conventions.
* [ ] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [ ] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [ ] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [ ] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

File: strings/is_palindrome.py [hunk 1]
```diff
@@ -14,15 +13,15 @@ def is_palindrome(s: str) -> bool:
...
-    s = "".join([character for character in s.lower() if character.isalnum()])
+    # Since punctuation, capitalization, and spaces are often ignored while checking
+    # palindromes,  we first remove them from our string.
+    s = "".join(character for character in s.lower() if character.isalnum())
     return s == s[::-1]
 
 
 if __name__ == "__main__":
-    s = input("Enter string to determine whether its palindrome or not: ").strip()
+    s = input("Please enter string to see if it is a palindrome: ")
```
Comment (CaedenPH) [inline strings/is_palindrome.py:23]:
```suggestion
    s = input("Please enter a string to see if it is a palindrome: ")
```

File: strings/is_palindrome.py [hunk 2]
```diff
@@ -14,15 +13,15 @@ def is_palindrome(s: str) -> bool:
     >>> is_palindrome("Mr. Owl ate my metal worm?")
     True
     """
-    # Since Punctuation, capitalization, and spaces are usually ignored while checking
-    # Palindrome,  we first remove them from our string.
-    s = "".join([character for character in s.lower() if character.isalnum()])
+    # Since punctuation, capitalization, and spaces are often ignored while checking
+    # palindromes,  we first remove them from our string.
```
Comment (CaedenPH) [inline strings/is_palindrome.py:17]:
```suggestion
    # palindromes, we first remove them from our string.
```

**Review by CaedenPH — APPROVED** [review]
Looks good to me

**Comment by cclauss** [comment]
@dhruvmanila @poyea

**Review by dhruvmanila — APPROVED** [review]

**Pull request merged**